pkg/view/component/base.go

```diff
@@ -88,6 +88,8 @@ const (
 	TypeYAML = "yaml"
 	// TypeIcon is a Icon component.
 	TypeIcon = "icon"
+	// TypeSingleButton is a SingleButton component.
+	TypeSingleButton = "singleButton"
```

Comment:
Just `button`. `ButtonGroup` is made up of `Button` componoents. A single button should be just `button` 

- `NewButton`
- `TypeButton = "button"`

Reply:
Got it, I'm gonna do the fix right now

Reply:
@wwitzel3 now I remember why I didn't call the component just `button` (I thought about it too),

The GroupButtonComponent files are already called for example `button.go`, so that's why I came up with the idea of calling it `SingleButton` to avoid collisions,  so I'm gonna try to make the change of the GroupButtonComponent files and renamed them to `button_group.go`  so we can have a better naming.